modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -278,13 +278,165 @@ definition {
 			value1 = "Amortiguador");
 	}
 
-	@description = "This is a test for COMMERCE-5805. UseSearchBarAndPagination"
+	@description = "COMMERCE-5802. Use a filter always check that the results are correct"
 	@ignore = "true"
 	@priority = "3"
+	test UseFilters {
+		property portal.acceptance = "false";
+
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Channels");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewChannelsEntry(
+			channelCurrency = "US Dollar",
+			channelName = "Test Channel",
+			channelType = "Site");
+
+		Button.clickSave();
+
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		Click(
+			key_categoryFilter = "Category",
+			locator1 = "CommerceEntry#FILTER_SELECT_CATEGORY");
+
+		Check.checkNotVisible(
+			checkboxName = "Brake System",
+			locator1 = "Checkbox#ANY_CHECKBOX");
+
+		Click(
+			key_menuItem = "Add Filter",
+			locator1 = "CommerceEntry#ANY_MENU_ITEM");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		AssertElementPresent(
+			key_entryName = "Premium Brake Fluid",
+			locator1 = "CommerceEntry#TABLE_LIST_TITLE");
+
+		Button.click(button = "Reset Filters");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		Click(locator1 = "CommerceEntry#FILTER_BACK_ARROW");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		Click(locator1 = "CommerceEntry#FILTER_DROPDOWN_MENU");
+
+		Click(
+			key_categoryFilter = "Catalog",
+			locator1 = "CommerceEntry#FILTER_SELECT_CATEGORY");
+
+		Click(
+			key_fieldType = "Test",
+			locator1 = "FormViewBuilder#FIELD_LABEL_MULTIPLE_SELECTION");
+
+		Click(
+			key_menuItem = "Add Filter",
+			locator1 = "CommerceEntry#ANY_MENU_ITEM");
+
+		AssertElementPresent(
+			key_entryName = "Hoses",
+			locator1 = "CommerceEntry#TABLE_LIST_TITLE");
```

Comment:
"Hoses" belongs to Minium Catalog, so if we want to use it we should assert it's not present in Master catalog

Reply:
In progress